src/test/java/org/openrewrite/java/testing/junit5/JUnit5MigrationTest.java

```diff
@@ -157,9 +157,76 @@ void upgradeMavenPluginVersions() {
         );
     }
 
+    @Test
+    void excludeJunit4Dependency() {
+        // Just using play-test_2.13 as an example because it appears to still depend on junit.
```

Comment:
I actually wanted to provide a pom with the junit dependency in the test, and then run it is:
```java
        rewriteRun(
          mavenProject("depends-on-junit", pomXml(remoteDepWithJunit, SourceSpec::skip)),
          mavenProject("needs-exclusion", pomXml(before, after)));
```
but somehow it works – the exclusion is added as intended – but it also gets a `MavenDownloadException` added in the output.

I was trying to do the same as other Maven tests that use `SourceSpec::skip`, but they all seem to do that for a parent, not a dependency.